pkg/skaffold/build/docker/docker.go

```diff
@@ -106,6 +106,12 @@ func (b *Builder) dockerCLIBuild(ctx context.Context, out io.Writer, name string
 	if err != nil {
 		return "", fmt.Errorf("unable to evaluate build args: %w", err)
 	}
+	for i, cliFlag := range a.CliFlags {
```

Comment:
Is this something that can be moved to where the other cli args are built: https://github.com/GoogleContainerTools/skaffold/blob/main/pkg/skaffold/docker/image.go#L629? I believe that would also add support for the GCB case.

Reply:
Sorry for the late reply. I fixed it!